server/svix-server/migrations/20240806212329_add_event_type_deprecated.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE eventtype ADD COLUMN deprecated BOOLEAN DEFAULT FALSE NOT NULL;
```

Comment:
We haven't added a new migration to the OSS version in quite a long time, is it fine to add one here?